docs/content/manual/pre-release/node/node-drain-deletion.md

```diff
@@ -65,3 +65,39 @@ NOTE: We might still 1 or 2 times of the pod's disruption budget message, but wi
 evicting pod longhorn-system/instance-manager-r-xxxxxxxx
 error when evicting pods/"instance-manager-r-xxxxxxxx" -n "longhorn-system" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
 ```
+
+
+## Stopped replicas on deleted nodes should not be counted as healthy replicas when draining nodes
+
+When draining a node, the node will be set as unscheduled and all pods should be evicted.
+
+But the replica will only be evicted if it has another healthy replica on the running node.
+
+##### Related Issue:
+- https://github.com/longhorn/longhorn/issues/2237
+
+##### Prerequisites:
+ - Start with 2 nodes cluster: Node_1, Node_2
+ - Double check that "Allow Node Drain with the Last Healthy Replica" is unchecked or return **false** with following command:
+    - `kubectl get settings.longhorn.io/allow-node-drain-with-last-healthy-replica -n longhorn-system`
+
+##### Steps:
+1. Create a 5Gi detached volume with 2 replicas.
+2. Stop Node_1 that contains one of the replicas.
+3. Attempts to drain Node_2 that contains remaining replica.
+    ```shell
+    kubectl drain <Node name> --delete-emptydir-data=true --force=true --grace-period=-1 --ignore-daemonsets=true --timeout=<Desired timeout in secs>
+    ```
+4. The node become cordoned. All pods on the cordoned node are evicted except the replica instance manager pod. And the message like below keeps appearing.
+    ```
+    evicting pod longhorn-system/instance-manager-r-xxxxxxxx
+    error when evicting pods/"instance-manager-r-xxxxxxxx" -n "longhorn-system" (will retry after 5s): Cannot evict pod as it would violate the pod's disruption budget.
+    ```
+5. Enable "Allow Node Drain with the Last Healthy Replica" and check return **true** with following command:
```

Comment:
nit: looks like there are 2 cases you are testing. Suggest breaking them up.
- Stopped replicas on deleted nodes should not be counted as healthy replicas when draining nodes
- Setting `Allow Node Drain with the Last Healthy Replica`